scripts/helm_handle_chart.sh

```diff
@@ -16,7 +16,8 @@ echo "BITOPS_CONFIG_COMMAND: $BITOPS_CONFIG_COMMAND"
 source "$BITOPS_SCHEMA_ENV_FILE"
 
 # Check for helm skip deploy condition
-if [ "$HELM_SKIP_DEPLOY" == "True" ]; then
+SANITIZED_HELM_SKIP_DEPLOY=$(echo "$HELM_SKIP_DEPLOY" | tr '[:upper:]' '[:lower:]')
+if [ "$SANITIZED_HELM_SKIP_DEPLOY" == "true" ]; then
```

Comment:
This works for `$HELM_SKIP_DEPLOY`, but doesn't scale well as there are many ENV vars in the script, and doing it only for one var but leaving others broken is not consistent. Also, there might be other plugins having the same issue.


Just look below, there are many such vars in this script:
https://github.com/bitops-plugins/helm/blob/a7631797ee8d36cdeba37b277defc2048f8993e4/scripts/helm_handle_chart.sh#L47

The ideal approach would be:
Creating a bash function, list all the boolean-type ENV vars and apply a lowercase function for all of them. 
For scaling out, because this function should be common for all the plugins, we can create a repo with bash libs we can share via git submodule with all the plugins. It may be a home for other reusable helpers like color support, different logging levels, casting `0`,`1`,`yes`,`no` to "bool" string and so on.

The first approach with a filter for a single var is not enough as we fix bug only for one var.
The second approach is too much for the current PR scope.
----

For now the pragmatic way I think is just to change `True` -> `true` in the code and call it done. Considering the level of current adoption and the fact that lowercase was documented it should be OK.
We just need to keep in mind to use lowercase `true`/`false` in the future where possible.

For the future we can document an enhancement creating a shared repo with bash helpers.

@PhillypHenning WDYT?

Reply:
I agree and have already applied the same sanity check for the SKIP_DEPLOY option across all plugins. I agree that the scope of this PR isn't enough to make the large scale change you're mentioning. 

Finally - this is a change that only needs to be done and noted ON OUR SIDE. From the client perspective, they shouldn't known about the sanitation we do. I.E this doesn't need doc updates